sources/tests/Unit/Inspector/Inspector.php

```diff
@@ -58,134 +45,150 @@ protected function getFixture()
+
         $fields_info = $this->getInspector()->getTableFieldInformation($this->getTableOid('with_simple_pk'));
-        $this
-            ->array($fields_info->slice('type'))
-            ->isIdenticalTo(['int4', 'inspector_test._someone', '_timestamptz'])
-            ;
+        $this->array($fields_info->slice('type'))
+            ->isIdenticalTo(['int4', 'inspector_test._someone', '_timestamptz']);
     }
 
-    public function testGetPrimaryKey()
+    /** @throws FoundationException */
+    public function testGetPrimaryKey(): void
     {
         $inspector = $this->getInspector();
-        $this
-            ->array($inspector->getPrimaryKey($this->getTableOid('no_pk')))
+        $this->array($inspector->getPrimaryKey($this->getTableOid('no_pk')))
             ->isEmpty()
             ->array($inspector->getPrimaryKey($this->getTableOid('with_simple_pk')))
             ->isIdenticalTo(['with_simple_pk_id'])
             ->array($inspector->getPrimaryKey($this->getTableOid('with_complex_pk')))
-            ->isIdenticalTo(['another_id', 'with_complex_pk_id'])
-            ;
+            ->isIdenticalTo(['another_id', 'with_complex_pk_id']);
     }
 
-    public function testChangePrimaryKey()
+    /** @throws FoundationException */
+    public function testChangePrimaryKey(): void
     {
         $inspector = $this->getInspector();
-        $this
-            ->array($inspector->getPrimaryKey($this->getTableOid('no_pk')))
+        $this->array($inspector->getPrimaryKey($this->getTableOid('no_pk')))
             ->isEmpty()
             ->array($inspector->getPrimaryKey($this->getTableOid('with_simple_pk')))
             ->isIdenticalTo(['with_simple_pk_id_renamed'])
             ->array($inspector->getPrimaryKey($this->getTableOid('with_complex_pk')))
-            ->isIdenticalTo(['another_id_renamed', 'with_complex_pk_id'])
-        ;
+            ->isIdenticalTo(['another_id_renamed', 'with_complex_pk_id']);
     }
 
-    public function testGetSchemaOid()
+    /** @throws FoundationException */
+    public function testGetSchemaOid(): void
     {
-        $this
-            ->integer($this->getInspector()->getSchemaOid('inspector_test'))
+        $this->integer($this->getInspector()->getSchemaOid('inspector_test'))
             ->variable($this->getInspector()->getSchemaOid('whatever'))
-            ->isNull()
-            ;
+            ->isNull();
     }
 
-    public function testGetSchemaRelations()
+    /** @throws FoundationException */
+    public function testGetSchemaRelations(): void
     {
-        $tables_info = $this
-            ->getInspector()
-            ->getSchemaRelations($this
-            ->getInspector()
-            ->getSchemaOid('inspector_test')
-        );
+        $tables_info = $this->getInspector()
+            ->getSchemaRelations($this->getInspector()
+                ->getSchemaOid('inspector_test')
+            );
```

Comment:
```suggestion
        $tablesInfo = $this->getInspector()->getSchemaRelations($this->getInspector()->getSchemaOid('inspector_test'));
```
change to CamelCase `$tables_info`